cross/toolchain.cmake

```diff
@@ -58,8 +62,13 @@ endmacro()
 
 # Specify link flags
 add_compile_param(CROSS_LINK_FLAGS "--sysroot=${CROSS_ROOTFS}")
-add_compile_param(CROSS_LINK_FLAGS "--gcc-toolchain=${CROSS_ROOTFS}/usr")
-add_compile_param(CROSS_LINK_FLAGS "--target=${TOOLCHAIN}")
+if (CLR_CMAKE_COMPILER STREQUAL "Clang")
+  message(WARNING "hello clang linker settings")
+  add_compile_param(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} --gcc-toolchain=${CROSS_ROOTFS}/usr")
+  add_compile_param(CROSS_LINK_FLAGS "${CROSS_LINK_FLAGS} --target=${TOOLCHAIN}")
+  message(WARNING $CROSS_LINK_FLAGS)
```

Comment:
You need to enclose the variable name in {} to get its value -> ${CROSS_LINK_FLAGS}

Reply:
I see, I'll push on the next rev. I suspect that the changes at line 67 and 68 will fix the issue.